app/src/main/java/com/github/sdp/tarjetakuna/ui/webapi/WebApiViewModel.kt

```diff
@@ -33,20 +31,13 @@ open class WebApiViewModel : ViewModel() {
         _apiError.value = Pair(resId, s)
     }
 
-    // Cards and sets live data
```

Comment:
why removing this ? how are we gonna get the result from the webapi in other part of the application now ?

Reply:
True, changes has been done.